fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -68,8 +69,11 @@ public class TaskRun implements Comparable<TaskRun> {
 
     private ExecuteOption executeOption;
 
+    private final String uuid;
```

Comment:
what's this used for ?

Reply:
It's to mark each different TaskRun which maybe triggered by Manual or ASYNC or PERIOD, otherwise you can not distinguish which task run it's.